### PR TITLE
Add a Pull Request labeler to Julia packages on GitHub(GCI)

### PR DESCRIPTION
**Pull request by FrancisJ000**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+documentation:
```
Comment (logankilpatrick) [inline .github/labeler.yml:1]: Note that the lables in the labeler.yml file correspond to the labels on the actual repo found here: https://github.com/JuliaPlots/PlotThemes.jl/labels. You can see that there is not currently a "documentation"  label. You should open a issue on this repo and request that label!

**Comment by logankilpatrick** [comment]
Also, congrats on your first pull request!

**Review by logankilpatrick — APPROVED** [review]
Awesome job! Should work like a charm once #38 is done.

**Comment by daschw** [comment]
Thanks a lot!

**Pull request merged**